src/components/select/multi-select/multi-select.spec.tsx

```diff
@@ -1099,6 +1100,37 @@ describe("MultiSelect", () => {
     });
   });
 
+  describe("isOptional", () => {
+    it("input box does not have required attribute", () => {
+      const propWrapper = mount(
+        <MultiSelect name="testSelect" id="testSelect" isOptional label="label">
+          <Option value="opt1" text="red" borderColor="red" fill />
+          <Option value="opt2" text="green" borderColor="green" />
+          <Option value="opt3" text="blue" />
+        </MultiSelect>
+      );
+      const input = propWrapper.find("input");
+      expect(input.prop("required")).toBeFalsy();
+    });
```

Comment:
comment (non-blocking): this might be a bit more readable with a blank line between each `it` block